tests/infrared/metrics-collectd-qdr.yaml.template

```diff
@@ -4,28 +4,47 @@ tripleo_heat_templates:
 
 custom_templates:
     parameter_defaults:
-        CollectdAmqpInterval: 1
-        CollectdDefaultPollingInterval: 1
-        CollectdExtraPlugins:
+        CollectdAmqpInterval: 5
+        CollectdDefaultPollingInterval: 5
+        CollectdDefaultPlugins:
             - cpu
             - df
+            - disk
             - hugepages
+            - intel_rdt
+            - interface
+            - ipmi
+            - load
+            - memory
             - ovs_events
             - ovs_stats
-            - load
+            - processes
+            - unixsock
             - uptime
         DnsServers: ["10.19.42.41","10.11.5.19","10.5.30.160"]
         ExtraConfig:
-            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::cpu::interval: 5
             collectd::plugin::cpu::reportbycpu: true
             collectd::plugin::cpu::reportbystate: true
             collectd::plugin::cpu::reportnumcpu: false
+            collectd::plugin::cpu::valuespercentage: true
+            collectd::plugin::df::interval: 300
             collectd::plugin::df::ignoreselected: true
+            collectd::plugin::df::fstypes: ['xfs']
+            collectd::plugin::disk::interval: 5
+            collectd::plugin::hugepages::interval: 60
+            collectd::plugin::intel_rdt::interval: 5
+            collectd::plugin::interface::interval: 5
+            collectd::plugin::ipmi::interval: 60
+            collectd::plugin::load::interval: 60
             collectd::plugin::load::reportrelative: true
-            collectd::plugin::interface::ignoreselected: true
+            collectd::plugin::memory::interval: 5
+            collectd::plugin::processes::interval: 5
+            collectd::plugin::uptime::interval: 60
+            collectd::plugin::virt::interval: 5
             collectd::plugin::virt::connection: "qemu:///system"
-            collectd::plugin::virt::hostname_format: "hostname uuid"
             collectd::plugin::virt::extra_stats: "cpu_util disk disk_err pcpu job_stats_background perf vcpupin"
+            collectd::plugin::virt::hostname_format: "hostname uuid"
```

Comment:
If you want these only in the Compute nodes, you can add a ComputeExtraConfig section.

ExtraConfig gets applied to all roles, but you can use <Rolename>ExtraConfig to apply only to the given role.

Reply:
Discussion on another channel has led to the following:
- The inclusion of `collectd::plugin::virt::*` in ExtraConfig appears to enable the virt plugin even without it being in the plugins list, without breaking the deploy. There are errors in the log but it seems to work fine. This needs to be re-tested from scratch to be sure.
- We think that putting `tripleo.collectd.plugins.collectd: virt` in a `ComputeExtraConfig` stanza is how we'd get it enabled only on Compute nodes. Not tested.

Reply:
I think the latter is the way to go.

Reply:
Does TripleO run the `ExtraConfig` first (apply to all nodes) and then the `<role>ExtraConfig` run after? If so then we could do both so that we set the preferred default, then provide an override for `<role>` configs.